packages/expo-auth-session/src/AuthSession.ts

```diff
@@ -127,7 +127,7 @@ export const getDefaultReturnUrl = sessionUrlProvider.getDefaultReturnUrl;
  * // Web: https://localhost:19006/redirect
  * ```
  *
- * @deprecated Use `makeRedirectUri({ path, useProxy })` instead.
```

Comment:
Hmm, `makeRedirectUri` still uses `useProxy` param:

https://github.com/expo/expo/blob/6fe8501665eb22c1f35c636e75298f245cec570d/packages/expo-auth-session/src/AuthSession.ts#L177

Maybe we should add `console.warn`  in dev mode with deprecation warning, when this param has been used? We do the similar thing already in:

https://github.com/expo/expo/blob/6fe8501665eb22c1f35c636e75298f245cec570d/packages/expo-auth-session/src/AuthSession.ts#L56-L60

Reply:
Thanks for the tip! I'll create a separate PR to follow up on that.